velox/common/config/SpillConfig.h

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+#include <stdint.h>
+#include <string.h>
+
+#include <folly/executors/CPUThreadPoolExecutor.h>
+#include "velox/common/compression/Compression.h"
+
+namespace facebook::velox::common {
+// Specifies the config for spilling.
```

Comment:
I assume there is no changes on this structure? Thanks!

Reply:
right, just moved to another namespace